utiliti/src/de/gurkenlabs/utiliti/swing/AssetPanelItem.java

```diff
@@ -166,13 +156,20 @@ public void mouseClicked(MouseEvent e) {
               e.consume();
             }
           }
+          @Override
+          public void mouseReleased(MouseEvent e) {
+            requestFocus();
+            if (e.getClickCount() == 2 && SwingUtilities.isLeftMouseButton(e) && addEntity()) {
+              e.consume();
+            }
+          }
         });
 
-    textField = new JTextArea();
-    textField.setWrapStyleWord(true);
-    textField.setRows(2);
-    textField.setLineWrap(true);
-    textField.setColumns(10);
+    SimpleAttributeSet attributes = new SimpleAttributeSet();
+    StyleConstants.setAlignment(attributes, StyleConstants.ALIGN_CENTER);

```

Comment:
Using justify will more or less look exaclty the same as the not aligned text before.
I'll remove the alignment and go back to a JTextArea, which is easier to use without the alignment styling.
I'll keep the 2 lines and button size changes.

Reply:
This won't center the text if it is a single line, which is still desirable in my opinion. Though I generally dislike the idea of using a `JTextPane` to display the filename as it is quite heavy. Maybe we can use a `JLabel` with html content and have the rename action sit inside a context menu with a separate popup?

Reply:
I changed the alignment back to before (left) and went back to a JTextArea instead of the JTextPane.
![image](https://user-images.githubusercontent.com/3458701/131249927-a371da73-f803-41d2-a73b-630f3c9ad74b.png)
To keep the longer names more visible, I kept the following behaviour where a long asset name is using the full available height until the asset is selected. Then the buttons are superimposed.
![image](https://user-images.githubusercontent.com/3458701/131249967-0c7ebca8-1f33-4900-870f-d5051f297d19.png)
![image](https://user-images.githubusercontent.com/3458701/131249974-0530680a-0f22-4c47-ba7b-4a48210c76f9.png)